internal/config/config.go

```diff
@@ -57,6 +52,12 @@ func ReadConfig() JobConfiguration {
 		fmt.Println("Failed reading env file. Running in simulation mode, reading from environment variables")
 	}
 
+	// Parse LOG_LEVEL after .env file is loaded
+	logLevel := os.Getenv("LOG_LEVEL")
+	level := ParseLogLevel(logLevel)
+	jc["log_level"] = level.String()
```

Comment:
why `String()`? Shouldn't we store the `Level` object directly?

Reply:
added both!~